client/components/account-status/style.scss

```diff
@@ -0,0 +1,62 @@
+/** @format */
+
+.woocommerce-account-status-header {
+	display: flex;
+	flex-direction: row;
+}
+.woocommerce-account-status-header div.title {
+	font-size: 16px;
+}
+.woocommerce-account-status-header div {
+	padding-right: 16px;
+}
+.woocommerce-account-status-header-controls {
+	flex-grow: 3;
+	text-align: right;
+}
+.woocommerce-account-status-item {
+	align-items: center;
+	display: flex;
+	flex-direction: row;
+	font-style: normal;
+	font-weight: normal;
+	height: 2em;
+	padding-right: 16px;
+}
+.woocommerce-account-status-item-label {
+	color: $gray-900;
+	flex-grow: 0;
+	font-size: 13px;
+	line-height: 16px;
+	margin: 0 4px;
+	width: 130px;
+}
+.woocommerce-account-status-item-value {
+	color: $gray-700;
+	flex-grow: 3;
+	text-align: left;
+}
+.woocommerce-account-status-item-value {
+	margin-left: 1.6rem;
+
+	.account-status__info__green {
+		fill: $studio-green-50;
+		color: $studio-green-50;
+	}
+
+	.account-status__info__red {
+		fill: $studio-red-60;
+		color: $studio-red-60;
+	}
+
+	.account-status__info__yellow {
+		fill: $studio-yellow-30;
+		color: $studio-yellow-30;
+	}
+
+	svg {
+		margin: 0 0.2rem;
+		position: relative;
+		top: 4px;
+	}
```

Comment:
Looks like this styling might be reused from existing components.

Reply:
Not anymore, I guess, after all the alignment work to make it play nicely with different icon sizes.

Reply:
well, I think colors might be reused. And I assume only `top` property of svg will need to be customized

Reply:
True, I moved colors into `shared.scss` and made use of it in other places.